### DISCUSSION TITLE
VirtualBox 7.1 can't be install on Silicon/Arm

### DISCUSSION THREAD

**Discussion by woj-tek** [Homebrew/discussions#5598] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.20-79-gb78f4b9
ORIGIN: https://github.com/Homebrew/brew
HEAD: b78f4b9d3ff121f4b62afc4f4509e2f334426e1b
Last commit: 44 minutes ago
Core tap HEAD: 73e340fe2545aec0b3a93bc364ad2cb4489c2af2
Core tap last commit: 24 hours ago
Core tap JSON: 12 Sep 08:21 UTC
Core cask tap HEAD: 231caee654653d3ded804bb73bff0c01c62268b4
Core cask tap last commit: 25 hours ago
Core cask tap JSON: 12 Sep 08:21 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: nvim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.46.0 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
```
```


### Output of `brew doctor`

```shell
n/a
```

### Description of issue

VirtualBox 7.1 was just released and it's cask has been updated but... it still has hard requirement on x86 (https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/v/virtualbox.rb#L25) even though this version does support arm64.

I see that virtualbox@beta cask handles this: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/v/virtualbox@beta.rb

Would using beta cask as a template but with updated versions/hashes be ok or that would be too much of a difference?

### Relevant casks

virtualbox, virtualbox@beta

**Comment by cho-m** [comment, marked as answer]
Cask was updated around time of discussion post so may be fixed already: https://github.com/Homebrew/homebrew-cask/pull/185148

**Reply by woj-tek** [reply]
Oh, thank you! <3
